test/hotspot/jtreg/serviceability/sa/TestJhsdbJstackPrintVMLocks.java

```diff
@@ -79,7 +79,10 @@ public static void main(String[] args) throws Exception {
             }
             throw new RuntimeException("Not able to find lock");
         } finally {
-            theApp.getProcess().destroyForcibly();
+            if (theApp.getProcess() != null) {
```

Comment:
This a really nice and usefule improvement, however doesn't fix the failure. 
It is unclear why the process is null.  Some exception should have been thrown, but we don't see it. The exception from finally block override the real problem.
So after your fix the test should start failing with real exception.  If you can reproduce, investigate and fix the problem then you can make it in this PR.
Otherwise, make sense to rename summary to something like:
`Improve error handling TestJhsdbJstackPrintVMLocks.java 
`
and investigate the problem later when failure has more info

Reply:
Agree that this PR doesn't fix the failure of LingeredApp throw exception during the intialization.
I have run this test standalone many times and can't reproduce the same failure. Maybe the failure only occur when run with other tests.
The summary has been renamed according your advice.